index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article1 class="box">
+			<section class="section1">
+				<h2>Yellow Dunk Cards</h2>
+			</section>
+
+			<section class="section2">
+
+				<img src="http://www.solidbackgrounds.com/images/125x125/125x125-banana-yellow-solid-color-background.jpg">
+
+				<p class="discription">
+					waistcoat single origin coffe
+					Waistcoat kitsch and single-origin coffee anddreamcatcher deep v,B hoodie echo park bushwick yr blog. Mixtape ethical tattooed dreamcatcher bicycle rights try-hard vegan, seitan plaid. Artisan mixtape austin street art.
+				</p>
+			</section>
+
+			<section class="section3">
+
+				<p class="lable"> Available
+				</p>
+				<p class="big">specifications</p>
+
+				<p class="measurments">
+
+					size: 4"*4"<br>
+					Weight: 8 oz
+				</p>
+				<footer>
+					<p class="small">
+						Specification vaid until 04/01/201
+					</p>
+				</footer>
+
+				<p class="big">Pricing</p>
+				<p class="measurments">
+					1: $1.99<br>
+					2-50: $1.49<br>
+					51+: $0.99
+				</p>
```

Comment:
Requirement 7: The third section contains the product pricing. (Again, you'll want to redefine your sections)
7-i: This section should contain a header. (Change your `<p class='big'>` to a header.)
7-ii: This section should contain three block elements. (currently you've got the text you'll want to be elements as within a single element, break it out into three elements as described below)
7-iii: Each block element contains information about the price for different quantities.